app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -75,6 +77,15 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
             // continue with navigation
             e.target.disabled = false;
             e.target.click();
+
+            e.preventDefault();
+
+            removeUnloadProtection();
+
+            if (inPersonURL) {
+              window.location.href = inPersonURL;
```

Comment:
I feel like you could also use `setStepName` from the `UseHistoryParam` hook here as well for consistency.